README.md

```diff
@@ -85,57 +85,26 @@ const app = express();
 ### Next steps
 
 Once you configure your app, you can use this package to access the Shopify APIs.
-See the specific documentation in the [Features section](#features) for instructions on how to get API access tokens and use them to query the APIs.
+See the [reference documentation](./docs/reference/README.md) for details on all the methods provided by this package.
+
+See the specific documentation in the [Tutorials section](#tutorials) for high-level instructions on how to get API access tokens and use them to query the APIs.
 
 As a general rule, apps will want to interact with the Admin API to fetch / submit data to Shopify.
```

Comment:
This sentence felt a bit awkward. Curious as the reason for this caution?

Reply:
This is just to build context and introduce the guides section, and mention the admin API clients.